Cargo.toml

```diff
@@ -8,6 +8,6 @@ license = "BSD-3-Clause"
 description = "ASN.1 (DER) serializer for Rust."
 
 [dependencies]
-byteorder = "*"
+byteorder = ">= 0.5.0"
```

Comment:
FYI, these are the same:

```toml
[dependencies]
byteorder = "0.5.0"
```

and

```toml
[dependencies]
byteorder = "^0.5.0"
```

http://doc.crates.io/crates-io.html#using-cratesio-based-crates

> When considering “compatible” versions, 0.1 and 0.2 are not considered compatible, but 1.0 and 1.1 are for example. If no operator is specified, this is the default requirement (e.g. 1.3 is the same as ^1.3).